### PR TITLE
Added docs for Data Sources

### PR DESCRIPTION
**Pull request by aarishshahmohsin**
Docs for Data sources are added in this PR.

Issue: https://github.com/openmainframeproject/software-discovery-tool/issues/140

### CODE REVIEW

File: docs/Data_sources.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## 2. Using bin/package_build.py
+
+Users can generate their own data sources by running the `bin/package_build.py` script. This script retrieves the latest data from the canonical resources for all supported distributions. The exceptions are Ubuntu, RHEL, and SLES, for which the data is pulled from the [PDS/distro_data](https://github.com/linux-on-ibm-z/PDS/tree/master/distro_data) repository. The usage can be seen in the [Using data from PDS](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Installation.md#using-data-from-pds) section in the Installation.md file.
+
+## 3. Using Custom Data Sources
+
+Users can choose to ignore the project's data sources and scripts entirely and instead load their own `.json` files. This option may be useful if the tool is being used for architectures not supported by the project or if it's being used internally within an organization to search for software. The`.json` files will be added in the `distro_data/` directory.
```
Comment (pleia2) [inline docs/Data_sources.md:17]: Let's adjust the wording here, so it doesn't come off as negative. Something like:

"Users do not need to use the project's data sources or scripts, and can instead load their own..."

File: docs/Data_sources.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## 2. Using bin/package_build.py
+
+Users can generate their own data sources by running the `bin/package_build.py` script. This script retrieves the latest data from the canonical resources for all supported distributions. The exceptions are Ubuntu, RHEL, and SLES, for which the data is pulled from the [PDS/distro_data](https://github.com/linux-on-ibm-z/PDS/tree/master/distro_data) repository. The usage can be seen in the [Using data from PDS](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Installation.md#using-data-from-pds) section in the Installation.md file.
+
+## 3. Using Custom Data Sources
+
+Users can choose to ignore the project's data sources and scripts entirely and instead load their own `.json` files. This option may be useful if the tool is being used for architectures not supported by the project or if it's being used internally within an organization to search for software. The`.json` files will be added in the `distro_data/` directory. 
+
+The files should have the format:
```
Comment (pleia2) [inline docs/Data_sources.md:19]: I'd adjust this to say: "The files should have the following format, with the description field being optional."

File: docs/Data_sources.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+```
+[
+{"packageName": "name","version": "version"},
```
Comment (pleia2) [inline docs/Data_sources.md:22]: We have an optional description field as well :smile: and we can make it more readable by formatting the json in a way that's easier on the eyes:

{
        "packageName": "name",
        "version": "version",
        "description": "Software Description"
},

**Review by pleia2 — CHANGES_REQUESTED** [review]

**Comment by aarishshahmohsin** [comment]
@pleia2 I have made the changes.

**Review by pleia2 — APPROVED** [review]

**Pull request merged**